src/relayserver/RelayServer.ts

```diff
@@ -40,6 +40,18 @@ import { toBuffer } from 'ethereumjs-util'
 
 import Timeout = NodeJS.Timeout
 
+/**
+ * After EIP-150, every time the call stack depth is increased without explicit call gas limit set,
+ * the 63/64th rule is applied to gas limit.
+ * As we have to pass enough gas to a transaction to pass 'relayRequest.request.gas' to the recipient,
+ * and this check is at stack depth of 3, we have to oversupply gas to an outermost ('relayCall') transaction
+ * by approximately 1/(63/64)^3 times.
```

Comment:
yes, this the approximation, but its 1.05, not 1.1

Reply:
I can live with it. It is better to oversupply then to catch a forwarder revert.